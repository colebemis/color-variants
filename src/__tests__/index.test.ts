import colorVariants from '../index'

it('returns expected array of colors', () => {
  expect(
    colorVariants({
      base: '#f00',
      light: { steps: 4, lighten: 0.9, hueShift: 20, saturate: -0.1 },
      dark: { steps: 2, darken: 0.5, hueShift: -10, saturate: 0.5 },
    })
  ).toMatchSnapshot()
})

it('assigns default values', () => {
  expect(
    colorVariants({
      base: '#f00',
      light: { steps: 4 },
      dark: { steps: 2 },
    })
  ).toMatchSnapshot()
})
