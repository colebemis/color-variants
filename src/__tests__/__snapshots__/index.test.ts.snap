// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns expected array of colors 1`] = `
Array [
  "#FEEEE7",
  "#FCC2AF",
  "#FB8C76",
  "#FD4C3C",
  "#FF0000",
  "#BF0000",
  "#800000",
]
`;
